amgut/db/patches/0010.sql

```diff
@@ -0,0 +1,22 @@
+CREATE TABLE ag.external_survey ( 
+	external_survey_id   bigserial  NOT NULL,
+	external_survey      varchar  NOT NULL,
+	external_survey_description varchar  ,
+	CONSTRAINT pk_external_surveys PRIMARY KEY ( external_survey_id )
+ );
+
+CREATE TABLE ag.external_survey_answers ( 
+	survey_id            varchar  NOT NULL,
+	external_survey_id   bigint  NOT NULL,
+	pulldown_date        date  NOT NULL,
+	answers              json  NOT NULL,
+	CONSTRAINT pk_third_party_surveys PRIMARY KEY ( survey_id )
```

Comment:
What if there are multiple external surveys associated with a single american gut survey? I think this primary key needs to be expanded to include the `external_survey_id`.

Additionally, if the plan is to store multiple "versions" of a person's answers to an external survey (retrieved on different dates), then the `pulldown_date` should also be included in the primary key, as well. If that is *not* the plan, I'm confused why this table cannot be combined with the `external_survey` table.